### PR TITLE
Translation of the We_accept_Monero_en for the pt-br

### PR DESCRIPTION
**Pull request by nelsonrenan**

### CODE REVIEW

**Comment by lh1008** [comment]
The same here @nelsonrenan. Please add the title of the article you worked upon. Thanks :)

**Comment by lh1008** [comment]
Pinging @botoserelepe @netrik182 for review. If anyone of you can helps us out would be of great help :).

File: monero-outreach-docs/translations/pt-br/archive/we_accept_monero_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Gostaríamos de te convidar para se unir conosco nos esforços em perpetrar uma boa e fora de moda ‘campanha de resmungar’ com seus comerciantes favoritos o porquê deles ainda não aceitarem XMR. A mais impressionante, e a mais bem sucedida ‘campanha de resmungar’ de todos os tempos tinha que ter sido escrita por George Lois. Em 1982 as empresas de TV a cabo não tinham interesse em um canal de música 24 horas, eles não achavam que funcionaria. George Lois apareceu com uma ideia e lançou uma série de comerciais que tinham estrelas do rock dizendo aos espectadores para ligarem para suas empresas de TV a cabo local e dizer “Eu quero minha MTV” O resto é história, [youtu.be/vAyOcWpm3QU](https://youtu.be/vAyOcWpm3QU?t=71).
+
+A missão, você deveria escolher aceitá-la, é contatar seu comerciante favorito e perguntar a ele se ele considerou aceitar Monero, porque esse seria seu método de pagamento favorito. Ele poderia dizer sim, ele poderia dizer não, mas finalmente você os fez pensar sobre o assunto. Não será a última vez que eles ouvem sobre o Monero, mas pode ser a primeira.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/archive/we_accept_monero_pt-br.md:7]:
```suggestion
A missão, caso você opte por cumpri-la, é contatar seu comerciante favorito e perguntar a ele se ele considerou aceitar Monero, porque esse seria seu método de pagamento favorito. Ele pode dizer sim, ele pode dizer não, mas finalmente você os fez pensar sobre o assunto. Não será a última vez que eles ouvem sobre o Monero, mas pode ser a primeira.

```

File: monero-outreach-docs/translations/pt-br/archive/we_accept_monero_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+A missão, você deveria escolher aceitá-la, é contatar seu comerciante favorito e perguntar a ele se ele considerou aceitar Monero, porque esse seria seu método de pagamento favorito. Ele poderia dizer sim, ele poderia dizer não, mas finalmente você os fez pensar sobre o assunto. Não será a última vez que eles ouvem sobre o Monero, mas pode ser a primeira.
+
+No mercado a sabedoria popular diz que alguém precisa ver a mensagem sete vezes antes de tomar consciência. Essa é uma regra polegar para publicidade, mas nós não estamos esperando encorajar uma conversa com comerciantes. Uma boa conversa vale facilmente por sete visitas nos anúncios. Pesquisas têm descoberto que pessoas que conhecem uma organização são cinco vezes mais propensas a ter uma opinião favorável sobre ela. Em outras palavras, quanto mais você falar sobre Monero, mais as pessoas irão gostar.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/archive/we_accept_monero_pt-br.md:9]:
```suggestion
No mercado a sabedoria popular diz que alguém precisa ver a mensagem sete vezes antes de tomar consciência dela. Essa é uma regra geral para publicidade, mas nós não estamos esperando encorajar uma conversa com comerciantes. Uma boa conversa vale facilmente por sete visitas nos anúncios. Pesquisas têm descoberto que pessoas que conhecem uma organização são cinco vezes mais propensas a ter uma opinião favorável sobre ela. Em outras palavras, quanto mais você falar sobre Monero, mais as pessoas irão gostar.

```

File: monero-outreach-docs/translations/pt-br/archive/we_accept_monero_pt-br.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+No mercado a sabedoria popular diz que alguém precisa ver a mensagem sete vezes antes de tomar consciência. Essa é uma regra polegar para publicidade, mas nós não estamos esperando encorajar uma conversa com comerciantes. Uma boa conversa vale facilmente por sete visitas nos anúncios. Pesquisas têm descoberto que pessoas que conhecem uma organização são cinco vezes mais propensas a ter uma opinião favorável sobre ela. Em outras palavras, quanto mais você falar sobre Monero, mais as pessoas irão gostar.
+
+Além disso temos criado este folheto do tamanho de uma nota de dólar que você pode imprimir e guardar em sua carteira para quando você estiver andando pelo mundo. Esta peça foca nos três recursos do Monero que são mais importantes para os pequenos donos de negócios e os convida a aprender mais em nossas páginas [Serviços Monero para comerciantes](https://www.monerooutreach.org/merchants/).
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/archive/we_accept_monero_pt-br.md:11]:
```suggestion
Além disso temos criado este folheto do tamanho de uma nota de dólar que você pode imprimir e guardar em sua carteira para quando você estiver andando pelo mundo. Esta folha foca nos três recursos do Monero que são mais importantes para os pequenos donos de negócios e os convida a aprender mais em nossas páginas [Serviços Monero para comerciantes](https://www.monerooutreach.org/merchants/).

```

File: monero-outreach-docs/translations/pt-br/archive/we_accept_monero_pt-br.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+Você pode fazer o download do arquivo pronto para impressão em [WhenYouAcceptMonero-en-WalletBuckslip.pdf](https://static.monerooutreach.org/img/MoneroMerchants/WhenYouAcceptMonero-en-WalletBuckslip.pdf). Boa sorte e vá com Deus!
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/archive/we_accept_monero_pt-br.md:13]: I would remove "vá com Deus", as a personal choice not to involve religion with financial matters, but it is totally up to you :)

**Review by botoserelepe — CHANGES_REQUESTED** [review]
@lh1008 review done :)
@nelsonrenan good job! Very nice translation! I have just made some suggestions to change a few words.

**Review by lh1008 — APPROVED** [review]
Everything looks fine. Sorry I took so long to be approved.

**Pull request merged**